### DISCUSSION TITLE
CODIGOS

### DISCUSSION THREAD

**Discussion by Jerrusjrmeny44** [community/community#54473] [Accessibility]
cuál es más eficiente codigo objeto o codigo fuente... y como aplicarlo en tus proyectos..?

**Comment by ghostinhershell** [comment]
Hi @Jerrusjrmeny44, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

With that being said, here's an article from HubSpot detailing the differences between Object Code and Source Code: [What is the Difference Between Source Code and Object Code? [+Examples]](https://blog.hubspot.com/website/source-code-vs-object-code).

Hope that helps!
